changelog/generated/v1.31.0.md

```diff
@@ -0,0 +1,20 @@
+## v1.31.0
+
+### Changes
+
+- (ansible): Update the `quay.io/operator-framework/ansible-operator` base image to now use Ansible 2.15.0.  Removes pre-installed Python modules that are not required to build and run the `testdata/ansible/memcached-operator` sample. Updates Python to 3.9 to work with Ansible 2.15.0. ([#6483](https://github.com/operator-framework/operator-sdk/pull/6483))
+- (ansible/v1): Updates the `ansible/v1` plugin scaffolding to: - Remove the `community.kubernetes` collection - Update the `operator_sdk.util` collection from `v0.4.0` --> `v0.5.0` - Update the `molecule.yaml` files to work with `molecule v5.1.0`. ([#6483](https://github.com/operator-framework/operator-sdk/pull/6483))
+- (helm): Use informer to list helm secrets to improve performance. ([#6354](https://github.com/operator-framework/operator-sdk/pull/6354))
+- Updated Reference from "to_be_owned.namespace" to "to_be_owned.metadata.namespace " in the ansible reference documentation. ([#6409](https://github.com/operator-framework/operator-sdk/pull/6409))
+
+### Removals
+
+- (ansible): Remove the `quay.io/operator-framework/ansible-operator-2.11-preview` image. This image will no longer be built after Operator-SDK v1.30.0. ([#6483](https://github.com/operator-framework/operator-sdk/pull/6483))
+
+### Bug Fixes
+
+- Fix a bug where `olm install` command is failed fo "no-match" error.
+The output in this case is something like:
+```$ operator-sdk olm install --verbose ... FATA[0001] Failed to install OLM version "latest": failed to create CRDs and resources: no matches for kind "OLMConfig" in version "operators.coreos.com/v1"```. Now, in this case, operator-sdk tries to create the resource again, until it succeed (or until the timeout exceeded). ([#6490](https://github.com/operator-framework/operator-sdk/pull/6490))
+- Fix a bug where `run bundle` command does not copy all FBC manifests into the new catalog image if custom index-image is passed. ([#6512](https://github.com/operator-framework/operator-sdk/pull/6512))
```

Comment:
nit
```suggestion
```$ operator-sdk olm install --verbose ... FATA[0001] Failed to install OLM version "latest": failed to create CRDs and resources: no matches for kind "OLMConfig" in version "operators.coreos.com/v1"```. Now, in this case, operator-sdk tries to create the resource again, until it succeeds (or until the timeout exceeded). ([#6490](https://github.com/operator-framework/operator-sdk/pull/6490))
- Fix a bug where `run bundle` command does not copy all FBC manifests into the new catalog image if custom index-image is passed. ([#6512](https://github.com/operator-framework/operator-sdk/pull/6512))
```